PPM.py

```diff
@@ -765,23 +794,154 @@ def get_metadata(flipnote):
 			f.write(newline.join(meta).encode("UTF-8"))
 			f.close()
 	elif sys.argv[1] == "-oa":
-                regex = re.compile(sys.argv[2])
-                os.chdir(sys.argv[3])
-                for filename in os.listdir("."):
-                        epoch = time.mktime(time.struct_time([2000, 1, 1, 0, 0, 0, 5, 1, -1]))
-                        
-                        if not os.path.isfile(filename):
-                                print "Error!\nSpecified file doesn't exist!"
-                                sys.exit()
-                        
-                        filetype = "ppm" if filename[-3:] == "ppm" else "tmb"
-                        flipnote = TMB().ReadFile(filename) if filetype == "tmb" else PPM().ReadFile(filename, ReadFrames=False)
-                        if not flipnote:
-                                continue
+		regex = re.compile(sys.argv[2])
+		os.chdir(sys.argv[3])
+		for filename in os.listdir("."):
+			epoch = time.mktime(time.struct_time([2000, 1, 1, 0, 0, 0, 5, 1, -1]))
+			if not os.path.isfile(filename):
+				print "Error!\nSpecified file doesn't exist!"
+				sys.exit()
+				
+			filetype = "ppm" if filename[-3:].lower() == "ppm" else "tmb"
+			flipnote = TMB().ReadFile(filename) if filetype == "tmb" else PPM().ReadFile(filename, ReadFrames=False)
+			if not flipnote:
+				continue
 
-                        meta = get_metadata(flipnote)
-                        if regex.match(meta["Original author"]):
-                                print filename
-                        
+			meta = get_metadata(flipnote)
+			if regex.match(meta["Original author"]):
+				print filename
+
+	elif sys.argv[1] == "-e":
+		if not hasffmpeg:
+			print "Error!\nffmpeg is not installed."
+			sys.exit()
+		in_file = sys.argv[2]
+		out_file = sys.argv[3]
+		try:
+			sleep_time = int(sys.argv[sys.argv.index("--sleep")+1])
```

Comment:
--sleep and --speed is not mentioned in the help text

Reply:
Ah good spot, I forgot to mention those, thanks